### ISSUE TITLE
deb/rpm package for linux

### ISSUE THREAD

**Issue by long76** [JetpackDuba/Gitnuro#47]
**Is your feature request related to a problem? Please describe.**
More fastest work without virtual env. Smaller install size.

**Describe the solution you'd like**
Build for current and previous supported Linux OS version. Depend from installed in system JRE.

**Describe alternatives you've considered**
Vanille deb/rpm package/zip which in startup request user specify path to JRE (17)

**Additional context**
I have bad perfomance on IDEA installed from Snap/Flatpak. Install from zip give best perfomance because work with OS directly.

**Comment by JetpackDuba** [comment]
Won't happen any time soon. The JRE would be necessary for the initial setup and changing the JRE version it's something that 99% of the users don't care much about. However, the main reason is that I don't want to be messing around distro specific packaging when Flatpak is a solid option where I have control over the environment.

You can always use the JAR and create a desktop file like the one in this repo https://github.com/flathub/com.jetpackduba.Gitnuro . Updating will only require replacing the JAR.

**Comment by long76** [comment]
> Won't happen any time soon. The JRE would be necessary for the initial setup and changing the JRE version it's something that 99% of the users don't care much about. However, the main reason is that I don't want to be messing around distro specific packaging when Flatpak is a solid option where I have control over the environment.
> 
> You can always use the JAR and create a desktop file like the one in this repo https://github.com/flathub/com.jetpackduba.Gitnuro . Updating will only require replacing the JAR.

i see. thanks for answer! but in future why not?) now more priority has development functional of soft)

**Comment by JetpackDuba** [comment]
Mostly because if the community wants to do it, they are free to, but I won't. There are tons of distros and I firmely belive that a universal solution like flatpak is the right option. I've had issues with DEB and RPM files multiple times and fixing them has been quite annoying, something that hasn't happened with Flatpak. If the performance was bad as it is with Snap, I'd consider it but I haven't noticed any difference, even in slower machines.

It  would also be a hassle to test in additional environments, building and testing in 3 OS is already time consuming for a side project. That's why I decided to provide 2 versions for each OS, one with the binaries and one easy way to install it: Linux has Flatpak+JAR, MacOS brew+application and Windows Installer+Portable.

In other words, the community is free to package in whatever format they want but I won't give support to specific issues of that packaging (obviously I'll give support to issues of Gitnuro itself).
